python/ui/test/test_add_remove_elements.py

```diff
@@ -0,0 +1,41 @@
+from python.common.basebrowser import BrowserExample
+from python.ui.method.add_remove_page import AddRemove
+from python.ui.method.internet_home_page import Homepage
+
+
+class TestAddRemove:
+
```

Comment:
This may be an extra return, I cant remember the PIP standard on class spacing.

Reply:
Spacing is acceptable, since PIP suggests one space between class and method declarations.